source/sass/buyers-guide/views/category.scss

```diff
@@ -78,7 +97,7 @@
         font-weight: 700;
         margin-bottom: 15px;
 
-        @media screen and (min-width: 750px) {
+        @media screen and (min-width: $bp-md) {
```

Comment:
Same, category page doesn't seem to have any `.category-item-header` elements. Is this entire `.category-item-header { }` block old code to be deleted?

Reply:
it is, removing.